pom.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.jboss</groupId>
+        <artifactId>jboss-parent-mr-jar</artifactId>
+        <version>31</version>
+    </parent>
+
+    <groupId>io.thorntail.jdk-specific</groupId>
+    <artifactId>thorntail-jdk-specific</artifactId>
+    <version>1-SNAPSHOT</version>
+
+    <properties>
+        <jdk.min.version>9</jdk.min.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.jboss.modules</groupId>
+            <artifactId>jboss-modules</artifactId>
+            <version>1.8.7.Final</version>
```

Comment:
I suppose the reason to have this version (latest is 1.9.0) is to mirror that of WFLY which is a base for TT?
Or can you update jboss-modules in TT independently?